web/html/errors.js

```diff
@@ -1,29 +1,39 @@
 const errorCodes = new Map([
-    ["Wallet-0001", "That nickname is taken. Try Another"],
-    ["Wallet-0002", "Wrong password. Try again"],
-    ["Wallet-0003", "Error while retrieving that wallet. Try again"],
-    ["Wallet-1001", "Enter a wallet nickname"],
-    ["Wallet-1002", "Enter a wallet password"],
+    ["Wallet-01", "That nickname is taken. Try Another"],
+    ["Wallet-02", "Wrong guiModal. Try again"],
+    ["Wallet-03", "Error while retrieving that wallet. Try again"],
+    ["Wallet-06", "Enter a wallet nickname"],
+    ["Wallet-07", "Enter a wallet password"],
     ["Wallet-1003", "Error while creating your wallet. Try again"],
-    ["Wallet-2001", "Select a wallet to delete"],
+    ["Wallet-15", "Action stopped. No wallet added to Thyra"],
+    ["Wallet-09", "Select a wallet to delete"],
     [
-        "Wallet-2002",
+        "Wallet-10",
         "Error while deleting that wallet. Close all programs using this wallet and try again",
     ],
-    ["Wallet-3001", "Error while importing this wallet. Try again"],
+    ["Wallet-11", "Error while importing this wallet. Try again"],
     [
         "Wallet-4001",
         "Error while connecting all your wallets. Reconnect all your wallets and try again",
     ],
-
-    ["Domains-0001", "Error while looking for your domain names"],
-    ["Domains-0002", "Error while connecting your domain and smart contract’s address"],
-
-    ["WebCreator-0001", "Error while creating your website container"],
-    ["WebCreator-1001", "Impossible to read you ZIP file. Try again"],
-    ["WebCreator-1002", "The upload of your ZIP file failed. Try again"],
+    ["Wallet-16" , "A wallet with the same nickname already exists."],
+    ["Unknown-0001", "An unknown error occured. Please try again"],
```

Comment:
The error in the code should have a consistent format. It should be Wallet-WXYZ,with W,Y,X and Z being digits